### PR TITLE
Changing the 5k flag to has_raised_funds

### PR DESCRIPTION
**Pull request by LindsayYoung**
We are going to have to run more sql updates for this.

### CODE REVIEW

**Comment by LindsayYoung** [comment]
Running the schema updates as the tests run.

File: data/sql_updates/candidate_aggregates.sql [hunk 1]
```diff
@@ -107,3 +107,5 @@ create index on ofec_candidate_totals_mv_tmp (cycle);
 create index on ofec_candidate_totals_mv_tmp (is_election);
 create index on ofec_candidate_totals_mv_tmp (receipts);
 create index on ofec_candidate_totals_mv_tmp (disbursements);
+create index on ofec_candidate_totals_mv_tmp (has_raised_funds);
+create index on ofec_candidate_totals_mv_tmp (federal_funds_flag);
```
Comment (ccostino) [inline data/sql_updates/candidate_aggregates.sql]: Ah good catch!

**Comment by LindsayYoung** [comment]
The flags work for me except `/committee/{committee_id}/candidates/` I don't think they should be there anyway.

**Comment by ccostino** [comment]
Okay you're good to go with this, thanks @LindsayYoung!

**Pull request merged**